### ISSUE TITLE
Error in requirements.txt 

### ISSUE THREAD

**Issue by gregdiaz125** [Benjamin-Pasternak/Security_project#2]
When running the  command `install -r requirements.txt` there appears to be a conflict with defining the version of dnspython and dnspython3, i tried this out on the ilabmachine( ptyhon ) and my personal machine here is what it gave out 
![unknown](https://user-images.githubusercontent.com/49290336/144763663-ff177220-c664-4b36-b284-ea7cfe6b9795.png)

**Comment by Lisk39** [comment]
yeah thats an issue with the DNSPython library itself, DNSPython3 depends on DNSPython but it tries to download the wrong version of DNSPython(1.15.0) The only way to get around this is to uninstall DNSpython after installing DNSPython3 and then reinstall the correct version of DNSPython(2.1.0) manually

**Comment by gregdiaz125** [comment]
you should include that in the read me

**Comment by Benjamin-Pasternak** [comment]
You should confirm you are using the correct version of python (3.8<=) as is required from dnspython3. Additionally, you should clone the entire repo. Finally please use pip3 when installing the dependencies. pip is the default for python 2. and using just pip could cause this issue to arise.

**Comment by gregdiaz125** [comment]
I uninstalled all of requirements.txt and I tried to install it again( i am running this on the python lab machine) when I ran `pip3 install -r requirements.txt ` i got the same thing

**Comment by symxmyz233** [comment]
It may **not be a bug.**
